xbmc/video/GUIViewStateVideo.cpp

```diff
@@ -436,6 +436,26 @@ bool CGUIViewStateWindowVideoNav::AutoPlayNextItem()
   return CSettings::Get().GetBool("videoplayer.autoplaynextitem");
 }
 
+bool CGUIViewStateWindowVideoNav::JumpToFirstUnplayedItem()
+{
+  if (m_items.IsVideoDb())
+  {
+    NODE_TYPE NodeType = CVideoDatabaseDirectory::GetDirectoryChildType(m_items.GetPath());
+    switch (NodeType)
```

Comment:
Point taken. Originally I also had music videos included as well, but removed it as I don't have any in my library so couldn't test them.

I left the switch in just in case that (or anything else) got added back in.